test/e2e/wmco_test.go

```diff
@@ -48,3 +53,18 @@ func setupWMCOResources() error {
 	}
 	return nil
 }
+
+// wmcoVersion returns the version that should be annotated on each Windows node
+func wmcoVersion() (string, error) {
```

Comment:
nit: I'd have injected this value into the custom struct by calling `wmco version` command or something like that.

Reply:
I would rather we use `wmco version`. You can further validate by comparing with `git` if you want but `wmco version` should be the source of truth.